command/e2etest/automation_test.go

```diff
@@ -71,14 +70,14 @@ func TestPlanApplyInAutomation(t *testing.T) {
 		t.Fatalf("failed to read plan file: %s", err)
 	}
 
-	stateResources := plan.State.RootModule().Resources
-	diffResources := plan.Diff.RootModule().Resources
+	// stateResources := plan.Changes.Resources
+	diffResources := plan.Changes.Resources
 
-	if len(stateResources) != 1 || stateResources["data.template_file.test"] == nil {
-		t.Errorf("incorrect state in plan; want just data.template_file.test to have been rendered, but have:\n%s", spew.Sdump(stateResources))
-	}
-	if len(diffResources) != 1 || diffResources["null_resource.test"] == nil {
-		t.Errorf("incorrect diff in plan; want just null_resource.test to have been rendered, but have:\n%s", spew.Sdump(diffResources))
+	//if len(stateResources) != 1 || stateResources ["data.template_file.test"] == nil {
+	// 	t.Errorf("incorrect state in plan; want just data.template_file.test to have been rendered, but have:\n%s", spew.Sdump(stateResources))
+	// }
```

Comment:
Do we want to remove this commented code?

Reply:
🤦‍♀ yes, thank you!